packages/xod-project/src/constants.js

```diff
@@ -38,6 +38,7 @@ export const ERROR = {
   LOOPS_DETECTED: 'The program has a cycle path. Use xod/core/defer-* nodes to break the cycle',
   DATATYPE_INVALID: 'Invalid data type',
   IMPLEMENTATION_NOT_FOUND: 'No implementation for {patchPath} found.',
+  CPP_AS_ENTRY_POINT: 'Can`t use patch that is not implemented in XOD as an entry point',
```

Comment:
Improper apostrophe. Should be `’`. Also, suggest to shorten:

> Can’t use patch not implemented in XOD as entry point

(the article omitted deliberately)

Reply:
Done!